### PR TITLE
BUGZ-530: Rename 'System Sounds' to 'UI FX'

### PR DESCRIPTION
**Pull request by zfox23**
Implements [BUGZ-530](https://highfidelity.atlassian.net/browse/BUGZ-530).

### CODE REVIEW

File: interface/resources/qml/hifi/simplifiedUI/settingsApp/audio/Audio.qml [hunk 1]
```diff
@@ -126,7 +126,7 @@ Flickable {
                 Layout.preferredWidth: parent.width
                 Layout.preferredHeight: 30
                 Layout.topMargin: 2
-                labelText: "System Sound Volume"
+                labelText: "UI FX Volume"
```
Comment (kencooke) [inline interface/resources/qml/hifi/simplifiedUI/settingsApp/audio/Audio.qml]: You should probably also change the original (complicated?) UI to match

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19783/HighFidelity-Beta-PR15682-0ba849d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19783/HighFidelity-Beta-Interface-PR15682-0ba849d.dmg

File: interface/resources/qml/hifi/simplifiedUI/settingsApp/audio/Audio.qml (hunk 1 above)
Comment (zfox23) [reply interface/resources/qml/hifi/simplifiedUI/settingsApp/audio/Audio.qml]: OK, done.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19784/HighFidelity-Beta-PR15682-326d161.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19784/HighFidelity-Beta-Interface-PR15682-326d161.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19784/HighFidelity-Beta-PR15682-326d161.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19784/HighFidelity-Beta-Interface-PR15682-326d161.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19784/HighFidelity-Beta-PR15682-326d161.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19784/HighFidelity-Quest-Beta-PR15682-326d161.apk)

**Pull request merged**